CHANGELOG/v1.8.0-beta.0.md

```diff
@@ -0,0 +1,410 @@
+- e2e: Allow setting worker machine count to nil in ApplyClusterTemplateAndWait & ConfigCluster (#10388)
+- e2e: Bump autoscaler to v1.30.0 (#10510)
+- e2e: Drop support for INIT env variables in clusterctl upgrade test (#10609)
+- e2e: Enable ability to test pre-releases of kubernetes (#10412)
+- e2e: Enhance E2E Tests to Filter Out Extra Provider-Specific Metadata (#10715)
+- e2e: Ensure resourceVersions are stable (#10530)
+- e2e: Export more func in test/e2e/common.go (#10420)
+- e2e: Fix finalizers assertions (#10735)
+- e2e: Improve E2E tests for finalizers and ownerRefs (#10730)
+- e2e: Improve E2E ValidateFinalizers and ValidateOwnerRef (#10693)
+- e2e: Improve metadata assertions in ClusterClass rollout test (#10851)
+- e2e: Kind bump to v0.23.0 (#10610)
+- e2e: Add restConfigModifier Option to clusterProxy (#10832)
+- e2e: Check for metadata.yaml when resolving releases to not try to use unreleased versions + avoid retry on 404 (also in clusterctl) (#10618)
+- e2e: Dump resources before deleting extensionconfig (#10874)
+- e2e: Improve output for ValidateResourceVersionStable by using BeComparable instead of Equal (#10652)
+- IPAM: Add Ready condition failure reasons (#10660)
+- IPAM: Add spec.clusterName to IPAddressClaim (#10182)
+- KCP: Add compare util using go-cmp, modify webhooks & KCP controller (#10628)
+- Logging: Fix log keys, use upper case for logs (#10613)
+- Logging: Set terminationMessagePolicy to FallbackToLogsOnError for all managers (#10580)
+- Machine: Improve logs & conditions when machine is waiting for node (#10757)
+- Machine: Propagate timeout fields from MachineSet to Machine during Machine deletion (#10589)
+- MachineDeployment: Deprecate MachineDeployment revision management (#10855)
+- MachineDeployment: Log reason for MachineDeployment rollouts / MachineSet creations (#10688)
+- MachineHealthCheck: Deprecate MHC MaxUnhealthy & UnhealthyRange fields (#10853)
+- MachineHealthCheck: MachineHealthCheck should take Machine's InfraReady condition (#10718)
+- MachinePool: MinReadySeconds for machinepools (#9837)
+- MachineSet: Deprioritize unknown NodeHealthy conditions for deletion (#10763)
+- MachineSet: Ensure unhealthy machines get deletion priority (#10755)
+- ClusterClass/e2e: Add MP back to dualstack E2E test (#10135)
+- KCP/ClusterResourceSet: Do not update observed generation if there are reconcile errors (#10736)
+- KCP/MachineSet: Flag for old infra machine naming (#10576)
+- Machine/MachineSet: Remove redundant watch event handlers via `Owns()` (#10048)
+- MachineSet/MachineDeployment: Preserve finalizers during MS/Machine reconciliation (#10694)
+- Release: Prepare main branch for v1.8 development (#10524)
+- Release: Push manifests for main & release-.* (#10521)
+- Release: Release notes also detect alpha releases as pre releases (#10369)
+- Release: Release v1.5.8 (#10438)
+- Release: Release v1.6.4 (#10439)
+- Release: Release v1.7.0 (#10440)
```

Comment:
This 3 should not show up. I would have expected them to be filtered out by the release not tool (I thought we implemented it already)

Reply:
I am checking why we are seeing these here.

Reply:
very strange for some reason the implementation did not work for these three PRs, I can see that it has been able to ignore all other release note PRs. I am checking what is different with these three PRs
cc @chandankumar4

Reply:
I thought that it might be because they have 🌱 in the title instead of 🚀  , but this release note PR(https://github.com/kubernetes-sigs/cluster-api/pull/10395) is ignored and it also has 🌱 in the title. So that's not the reason something else is different there.

Reply:
I’ll look it more closely and will open up an issue with detail.

Reply:
Thx for following-up! Just noticed it and was surprised.

I thought in general these PRs must have 🚀 in the title

Reply:
we started doing it after 1.7.0 ,